docs/ca/dev/monitoring/opentelemetry-instrumentation.md

```diff
@@ -0,0 +1,504 @@
+                       | {}              | A valid JSON object where keys represent service names and values define URL patterns.
+                                                                                                                                                              |
+| OTEL_DEFAULT_SERVICE_NAME          | If no service name is provided and `OTEL_SERVICE_NAME_MAPPING` is not defined, this default name is used.
+                                            | Default Service | any valid string                                                                                                                                                                                                              |
+| OTEL_BSP_SCHEDULE_DELAY            | Defines the delay in milliseconds before sending a batch of spans to the exporter. A higher value results in larger batches.
+                                      | 1000              | 0...100000000                                                                                                                                                                                                                 |
+| OTEL_BSP_MAX_QUEUE_SIZE            | Defines the maximum number of spans that can be queued for processing in a single request.
+           | 2048              | At least an amount of spans you want to see                                                                                                                                                                                   |
+| OTEL_BSP_MAX_EXPORT_BATCH_SIZE     | Defines the batch size for spans. Once this limit is reached, the batch is sent to the exporter.
+                                                                           | 512               | More than `0` and less than `OEL_BSP_MAX_QUEUE_SIZE`                                                                                                        |
+| OTEL_SDK_DISABLED                  | If set to `true`, no traces are generated or sent to the backend. The default value is `true`; change only after the collector is up and running.
+| true            | Can be a boolean or a string representation of a boolean, such as `true` or `false`.
+                                                                                                                                                                |
+| OTEL_PHP_DISABLED_INSTRUMENTATIONS | Allows disabling specific parts of additional instrumentation. For example, to exclude all Redis spans, set the value to `spryker_otel_redis`. Multiple instrumentation parts can be disabled by providing a comma-separated list.
+ |           | `spryker_otel_redis`, `spryker_otel_elastica`, `spryker_otel_propel`, `spryker_otel_rabbitmq`, `spryker_otel_guzzle`, or `all`. Also you can select a few of instrumentation, e.g. 'spryker_otel_rabbitmq,spryker_otel_propel' |
```

Comment:
What do you mean by `a few of instrumentation`?

Reply:
that you can select not one, but a few